Webui 543 ifloggedin (#559)
MichaelMure/git-bug: webui/Readme.md

## How to develop

### Run GraphQL backend

1. Download a git-bug stable binary.

2. Execute git-bug binary inside directory for the git repository it will manage issues:
   - git-bug webui -p 3001

### Run ReactJS front-end

1. Clone git-bug repository.

2. Enter webui directory and install libraries needed:
   - npm install

3. Generate ts code from graphql files and run webui in development mode

>    - npm start
>    - If You got compilation errors from lint, run lint command below and start again:
>       - npm run lint -- --fix
>       - npm start

Reviewer: ```suggestion
   - `make start` or `npm start`
   - If you get some lint errors, run the lint command below and start again:
      - `make fix-lint` or `npm run lint -- --fix`
      - `make start` or `npm start`
```
